MediaBrowser.Model/Entities/MetadataProvider.cs

```diff
@@ -7,6 +7,12 @@ namespace MediaBrowser.Model.Entities
     /// </summary>
     public enum MetadataProvider
     {
+        /// <summary>
+        /// This metadata provider is for users and/or plugins to override the
+        /// default merging behaviour.
+        /// </summary>
+        Custom = 0,
```

Comment:
Are we certain that 0 is unused? Could be 999 or something instead to avoid collisions

Reply:
I looked at the hit blame history, and only '1' was previously used.

Reply:
*git

Reply:
I can change it to either '20' (the next unused number) or '999' (your suggestion) if you don't want to use '0'

Reply:
It's fine